tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/WeakCipher/WeakCipherTests.cs

```diff
@@ -63,10 +63,10 @@ public void IntegrationDisabled(string variableName, string variableValue)
     {
         SetEnvironmentVariable("DD_IAST_ENABLED", "true");
         SetEnvironmentVariable(variableName, variableValue);
-        const int expectedSpanCount = 6;
+        const int expectedSpanCount = 1; // we actually expect 0 here
         using var agent = EnvironmentHelper.GetMockAgent();
         using var process = RunSampleAndWaitForExit(agent);
-        var spans = agent.WaitForSpans(expectedSpanCount, returnAllOperations: true);
+        var spans = agent.WaitForSpans(expectedSpanCount, 2_000, returnAllOperations: true); // wait 2 seconds - we expect no spans
```

Comment:
Just another case where we are actually expecting no spans

Reply:
Could remove the wait entirely I think 🙂